config/sara/sara.yml

```
@@ -5,7 +5,10 @@ version: v1.41
 base_path: ''
 
 # How many days an enrolled monitoree should be monitored
-monitoring_period_days: <%= ENV["MONITORING_PERIOD_DAYS"] || 14 %>
+monitoring_period_days: <%= ENV["MONITORING_PERIOD_DAYS"] || 10 %>
+
+# How many days an enrolled case should be monitored until recovery
+recovery_period_days: <%= ENV["recovery_period_days"] || 5 %>
```

Any reason this should be lowercase?
```suggestion
recovery_period_days: <%= ENV["RECOVERY_PERIOD_DAYS"] || 5 %>
```

---

good catch, this was done on accident when I performed a find and replace: https://github.com/SaraAlert/SaraAlert/pull/1188/commits/25a80f08890b849c9b61fe38fb529601853ab4ce